modules/key-vault/vault/.test/min/main.test.bicep

```diff
@@ -43,5 +43,6 @@ module testDeployment '../../main.bicep' = {
     name: '${namePrefix}${serviceShort}002'
     // Only for testing purposes
     enablePurgeProtection: false
+    enableRbacAuthorization: true
```

Comment:
min tests should only contain required parameters. I'd suggest to remove this and instead change the `enableRbacAuthorization` default value to `true` in the main.bicep template

```suggestion
```

Reply:
All suggestion committed, readme and main.json updated